### ISSUE TITLE
The table in the what's new page is to tight on small screens

### ISSUE THREAD

**Issue by anthonydillon** [canonical-web-and-design/vanilla-framework#4238]
**Describe the bug**

The key information on the what's new page is not accessible on small screens.

**To Reproduce**

1. Go to https://vanillaframework.io/docs/whats-new on a phone
2. See the version and other important information is truncated

**Expected behavior**
I should be able to see the all the information as it's key content.

**Screenshots**
![Screenshot_20220111-235118.jpg](https://user-images.githubusercontent.com/1413534/149039519-b6cd251a-bd75-4ff5-a55d-41d3869baad3.jpg)

**Additional context**

The status key below is easy to read and looks nice (if that helps at all)

**Comment by elioqoshi** [comment]
@lyubomir-popov @bartaz quick draft. Would it be structurally okay / possible to merge the status and version values as part of the component? They feel more like labels than needing their own column: ![grafik.png](https://images.zenhubusercontent.com/5a5649f54b5806bc2bc23b6d/688ebd27-635d-47ff-b281-8e53ce1bda8a)

Alternatively we could include them in the table from the start in this manner?

Anyhow, quick two options with 1  having the description underneath (not sure if this is possible as part of a table component or whether that's the right approach though).

2 merges status and version into the component field which I think could work out better.

**Comment by lyubomir-popov** [comment]
What's the point of the version column? We have one table per release, so you can remove the version column altogether.

**Comment by elioqoshi** [comment]
Good point. Not sure how it ended up there or whether it was automatical. Thought there might be some sub-version releases which might be relevant?

Regardless, even if that's remove there is still space lacking

**Comment by cristinadresch** [comment]
@bartaz any thoughts?

**Comment by bartaz** [comment]
@elioqoshi 

Overall I think version column is valuable, while we keep separate table for latest release, we do merge all previous releases into one long table, like here with 2.0:

<img width="846" alt="Screenshot 2022-02-02 at 12 00 57" src="https://user-images.githubusercontent.com/83575/152141731-5800e195-076a-4be5-96e8-a07f9fc12dc6.png">

It is valuable to have this version information. While not ideal, we probably can hide version on small screens (because it's definitely supplementary information).

Visually I like your proposal of having notes below rest of the data, but I don't think it would be easily achievable with HTML table now.

Second option to merge status label and version into one cell seems interesting compromise.

The issue is from implementation point of view we don't have much flexibility to change table layout on different screen sizes, so we would need to explore that.

**Comment by bartaz** [comment]
Triage: Low effort, low impact: As a quick fix we could apply the horizontal scroll as in other places in the docs.
